release_notes/20230420_brazilian.xml

```diff
@@ -53,7 +53,7 @@ Aqui estão alguns destaques das atualizações desde o aniversário anterior:
 [*][b]Tilarus-5: Arctic Infiltration:[/b] Corrigido uber drones aparecendo e desaparecendo logo após o início da missão.
 [*][b]Tilarus-5: SynTek Hospital:[/b] Volume do alarme aumentado.
 [*][b]Lana's Escape: Lana's Bridge:[/b] Otimizações de desempenho significativas para iluminação, renderização e localização de caminhos.
-[*][b]Lana's Escape: Lana's Sewer:[/b] Os barris radioativos na arena causam mais dano aos shamans.
+	[*][b]Lana's Escape: Lana's Sewer:[/b] Os barris radioativos na arena causam mais dano aos shamans.
```

Comment:
This should be two tabs and it should be the English version of the line (otherwise, translators have to look at multiple files).

Reply:
Roger, roger. I'll redo that.